services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfEventWfmTopology.java

```diff
@@ -54,9 +56,10 @@
     @VisibleForTesting
     public static final String DISCO_SPOUT_ID = "disco-spout";
     private static final String DISCO_BOLT_ID = OfeLinkBolt.class.getSimpleName();
-    private static final String TOPO_ENG_BOLT_ID = "topo.eng-bolt";
     private static final String SPEAKER_BOLT_ID = "speaker-bolt";
     private static final String SPEAKER_DISCO_BOLT_ID = "speaker.disco-bolt";
+    private static final String NETWORK_TOPOLOGY_BOLT_ID = "network-topology-bolt";
```

Comment:
`topology-bolt` should be enough.

or `persistence-bolt` ... 

PS `-bolt` suffix is excess.

(minor)